registration/views.py

```diff
@@ -403,9 +419,11 @@ def render_done(self, form, **kwargs):
         self.storage.reset()
         # also clean out the session
         clean_session(self.request.session)
-        clean_cobrand_session(self.request.session)
-        clean_widget_session(self.request.session)
-        clean_email_source_session(self.request.session)
+        # I Don't think we need to clear those session values on registration done, as it is going to be cleared on the
+        # next visit for Start page anyways.
+        # clean_cobrand_session(self.request.session)
+        # clean_widget_session(self.request.session)
```

Comment:
@waseem-omar even if someone comes in with bad data in their url? Are you sure that if someone came in via widget, then clicked on a shared URL that it would have only the correct data in the session?